dashboard/src/components/OperatorNew/OperatorNew.tsx

```diff
@@ -261,7 +263,7 @@ export default function OperatorNew() {
                 </div>
               </div>
               <div className="clr-form-control">
-                <CdsButton type="submit" disabled={disableInstall}>
+                <CdsButton type="submit" aria-disabled={disableInstall}>
```

Comment:
Why this change? From what I can see in a quick search, it seems both `aria-disabled`  and `disabled` actually disable the input... however, the `aria-` one does not prevent the element to be announced in a screen reader. Is this the motivation of the change? If so, perhaps we should extend it to the rest of the inputs, no?